src/serialize/root.zig

```diff
@@ -0,0 +1,133 @@
+const std = @import("std");
+const testing = std.testing;
+const Allocator = std.testing.allocator;
+
+const MAX_FIELD_COUNT: u8 = 255;
+
+pub fn tomlize(allocator: std.mem.Allocator, obj: anytype, writer: anytype) !void {
+    try serialize_struct(allocator, obj, writer);
+}
+
+fn serialize_struct(allocator: std.mem.Allocator, value: anytype, writer: anytype) !void {
```

Comment:
Let's use the current zig naming conventions: https://nathancraddock.com/blog/zig-naming-conventions/ -> function names should be in camelcase.

Reply:
Yeah sorry, I always have to juggle between a lot of languages and for some reason I always get Zig wrong. I will just fix it.